jmx-metrics/src/test/groovy/io/opentelemetry/contrib/jmxmetrics/OtlpIntegrationTests.groovy

```diff
@@ -4,24 +4,23 @@
  */
 package io.opentelemetry.contrib.jmxmetrics
 
+import io.opentelemetry.proto.common.v1.AnyValue
 import io.opentelemetry.proto.common.v1.InstrumentationLibrary
-import io.opentelemetry.proto.common.v1.StringKeyValue
+import io.opentelemetry.proto.common.v1.KeyValue
 import io.opentelemetry.proto.metrics.v1.InstrumentationLibraryMetrics
 import io.opentelemetry.proto.metrics.v1.Metric
-import io.opentelemetry.proto.metrics.v1.DoubleHistogram
-import io.opentelemetry.proto.metrics.v1.DoubleHistogramDataPoint
 import io.opentelemetry.proto.metrics.v1.ResourceMetrics
+import io.opentelemetry.proto.metrics.v1.Summary
+import io.opentelemetry.proto.metrics.v1.SummaryDataPoint
 import org.testcontainers.Testcontainers
 import spock.lang.Requires
-import spock.lang.Retry
 import spock.lang.Timeout
 import spock.lang.Unroll
 
 @Requires({
     System.getProperty('ojc.integration.tests') == 'true'
 })
 @Timeout(90)
-@Retry
```

Comment:
After being stuck for a long time I realized that this annotation is dangerous - it would retry on assertion failure, and eventually hit the Timeout, throwing a Timeout error while starting testcontainers and not showing anything about the assertion. Better if we can make the test bodies more robust, I did that in one place